frameworks/helloworld/tests/test_isolation.py

```diff
@@ -33,17 +33,21 @@ def configure_package(configure_security):
 
 
 @pytest.mark.sanity
-def test_once_task_does_not_restart_on_config_update():
+def test_():
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
     config.check_running(foldered_name)
 
     sdk_plan.wait_for_completed_deployment(foldered_name)
-    assert os.path.isdir('./tmp')
-
-
+    assert !os.path.isdir('./tmp')
```

Comment:
This doesn't do what you think, remember that the pytest code is executed on the machine executing the test and not in the tasks of the service.

You need to do a dcos task exec into the task container to check if there is a tmp folder or not. Like this:
```
code, _, _ = sdk_cmd.service_task_exec(config.SERVICE_NAME, "hello-0-server", "ls | grep tmp")

assert code > 0
```

Note: task exec will put you into the $MESOS_SANDBOX directory on any version of DC/OS > 1.10. DC/OS 1.9 will require a bit of fiddling.